spec/coverage_helper.rb

```diff
@@ -5,20 +5,19 @@ module CoverageHelper
 
   ##
   # Skip list used to avoid loading certain patterns within
-  # the logstash directories, this patterns are excluded becuause
+  # the logstash directories, this patterns are excluded because
```

Comment:
it was a conscious and deliberate change from me ... the typo was hurting my eyes :P